apps/total-typescript/src/pages/api/sanity/transcript/update.ts

```diff
@@ -0,0 +1,117 @@
+import {withSentry} from '@sentry/nextjs'
+import {NextApiRequest, NextApiResponse} from 'next'
+import sanityClient from '@sanity/client'
+import {json} from 'stream/consumers'
+import {uniqueId} from 'lodash'
+import {randomUUID} from 'crypto'
+
+const addTranscriptToVideoDocument = async (
+  req: NextApiRequest,
+  res: NextApiResponse,
+) => {
+  if (req.method === 'POST') {
+    const client = sanityClient({
```

Comment:
we have a configured client here https://github.com/skillrecordings/products/blob/main/apps/total-typescript/src/utils/sanity-client.ts